contentcuration/contentcuration/models.py

```diff
@@ -1156,13 +1156,14 @@ def get_details(self):
 
         # Add "For Educators" booleans
         for_educators = {
-            "coach_content": resources.filter(role_visibility=roles.COACH).exists(),
-            "exercises": resources.filter(kind_id=content_kinds.EXERCISE).exists(),
+            "coach_content": resources.filter(role_visibility=roles.COACH).count(),
```

Comment:
Note that count is a considerably more expensive query than exists - so if we don't absolutely need to show this, it could help reduce the expense of the endpoint somewhat.

Reply:
The count is part of the design for details, but could be worth revisiting the design if the load time is too long

Reply:
Fair enough - probably other optimizations we can do in the details query as a whole that make this a non-issue for now, anyway!